_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -61998,7 +61998,7 @@
 	},
 /area/engine/controlroom)
 "iTF" = (
-/obj/machinery/door/window/reinforced/reversed{
```

Comment:
This one is reversed unlike all the others, make sure this is the direction you want

Reply:
the reversed door doesn't even change dir

Reply:
Then why did we have a whole PR to do that... make sure that its the same in game as it is in the map editor just to be safe. Dunno why they would be the same